app/views/projects/show.html.slim

```diff
@@ -12,6 +12,11 @@
     h3 Configurations
     .d-flex.align-items-center
       = link_to "Create", new_project_deployment_configuration_path(@project), class: "btn btn-primary"
+
+  .mt-4
+    = link_to "/auth/slack?project_id=#{@project.id}" do
```

Comment:
Please extract string to some kind of slack router

Reply:
Created constants and moved slack link and github one

Reply:
I think it's better to create a router. I've already created router for github https://github.com/CleverLabs/deployqa/pull/68/files#diff-e8cd9d75b8fe59939094186c46af755f, so I think it's better to create one for slack for consistency

Reply:
Hm, that's a different kind of urls, my bad